docs/USB-Camera-Collection.postman_collection.json

```diff
@@ -665,6 +665,114 @@
 			},
 			"response": []
 		},
+		{
+			"name": "Get Pixel Format",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"  pm.response.to.have.status(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "http://{{command_host}}:{{command_port}}/api/v3/device/name/{{camera_name}}/PixelFormat?PathIndex={{path_index}}",
```

Comment:
@ChristianDarr-personal But for that I need to get all the related code which is still in review. I think it would be better to incorporate this after you merge?

Reply:
I don't think there is any of my code needed, if they merge at the same time, the only thing we need to adapt is the postman collection